webview/src/experiments/components/table/styles.module.scss

```diff
@@ -804,13 +748,18 @@ td {
 .experimentCellSecondaryName {
   color: $meta-cell-color;
   font-size: 0.75em;
-  display: flex;
-  align-items: center;
+
```

Comment:
Noticed that the text-overflow wasn't working. Fixed now:

<img width="189" alt="image" src="https://user-images.githubusercontent.com/43496356/231873747-218545a3-b3cd-4783-9d4f-a25809f8e179.png">